fake-delius-integration-api/mappings/user-info-api.json

```diff
@@ -0,0 +1,14 @@
+{
+    "request": {
+      "method": "GET",
+      "urlPathPattern": "/user/make-recall-decision-api details"
```

Comment:
Is this a valid pattern with the blank space? Or did you accidentally leave the `details` part in?

Reply:
Cannot recall but you are correct that it definitely doesn't look right. Will investigate.

Reply:
The more I look, the more I think this is a mistype that I just straight up missed amongst the everything else going on. Change coming in the next push.